website/docs/guides/markdown-features/markdown-features-react.mdx

```diff
@@ -507,11 +507,13 @@ By convention, using the **`_` filename prefix** will not create any doc page an
 This is text some content from `_markdown-partial-example.mdx`.
 ```
 
+<!-- prettier-ignore-start -->
 ```jsx title="someOtherDoc.mdx"
 import PartialExample from './_markdown-partial-example.mdx';
 
-<PartialExample name="Sebastien" />;
+<PartialExample name="Sebastien" />
 ```
+<!-- prettier-ignore-end -->
```

Comment:
Similar to: https://github.com/facebook/docusaurus/blob/46d705a487168aa1c09554466d81206ede2aa067/website/docs/guides/markdown-features/markdown-features-react.mdx?plain=1#L464-L471